### ISSUE TITLE
[Feat. req.] Add `-r --reset` to `cargo-install-update-config`

### ISSUE THREAD

**Issue by cyqsimon** [nabijaczleweli/cargo-update#168]
Correct me if I'm wrong, but I don't think there is currently a way to clear (or reset) the configuration for a package via `cargo install-update-config`. I know it's possible to edit `~/.cargo/.install_config.toml` directly, but then why have the config binary in the first place?

Anyways, it would be nice if I can just run `cargo install-update-config --reset <crate>` to clear all its settings. I feel like it's a nice quality-of-life addition.

**Comment by nabijaczleweli** [comment]
Yeah, that sounds sensible – the `master` branch (at least f59559695d00fb6e90fa60e15808d3eacb9228d9) has this p. much exactly as you described, try it out, if you'd.

**Comment by nabijaczleweli** [comment]
Released in [`v8.1.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v8.1.0)
